### ISSUE TITLE
508-defect-2 [FOCUS MANAGEMENT, SCREENREADER]: Focus on page load within the form SHOULD be placed on the H2 (Steps)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18965]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Within the BDD form flow, on page load, the focus is regularly placed on the `h1` (the title of the form), when it should be placed on the `h2` (the step the user is on). Placing the focus on the `h2` (the step) is better since it changes page by page and is relevant to the content being worked on.

Note here: this may require more thought into the distant future. GOV.uk handles focus by placing it on the _main question_ as opposed to the step or title of the form which are _not_ written as headings. 

## Point of Contact

**VFS Point of Contact:** Josh

## Acceptance Criteria

* [ ] Focus is placed on the `h2` step, not the `h1` _within_ the form

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Screenshots

https://user-images.githubusercontent.com/14154792/105762539-892ebb00-5f22-11eb-9154-e3300441ab7b.mov

Focus should go to "Step 1 of 5"

**Comment by Mottie** [comment]
I just merged this, so it'll be available on staging soon.

**Comment by joshkimux** [comment]
@Mottie Confirmed this has been updated! Great work!
